### DISCUSSION TITLE
Weird android animation on 3.21.0

### DISCUSSION THREAD

**Discussion by jslok** [software-mansion/react-native-screens#1792] [Q&A] [answered]
The screen push animation seems to be the old screen slides to the left halfway then the new screen fades in. Looks awkward. Is this correct or is something wrong with my setup? I was unable to run the example app.

**Comment by kkafar** [comment, marked as answer]
Android 13 (Android SDK 33+) introduced change of default animation and `react-native-screens` followed, however there are some reports that new default animation provided by `react-native-screens` does differ from the purely native one.
